db/schema.rb

```diff
@@ -209,6 +210,7 @@
     t.boolean  "locked",                      default: false
     t.boolean  "itunes_block",                default: false
     t.string   "feed_rss_alias"
+    t.string   "restrictions"
```

Comment:
Seems unlikely to cross 256 for this purpose...
maybe this should be a `text` column since it is json?

Reply:
Good point.  If we ever supported `type = uri` ... could hit that 256 pretty quick.